packages/fetcher/src/ccxt-fetcher.ts

```diff
@@ -28,14 +27,15 @@ export default class CCXTFetcher implements FetcherInterface {
    * Fetch price for a given pair.
    *
    * @param {string} pair
-   * @returns {Promise<string>} (bid + ask) / 2
+   * @returns {Promise<string>} vwap price
    * @memberof CCXTFetcher
    */
   getPrice(pair: string): Promise<string> {
     return this.exchange.fetchTicker(pair).then((ticker: Ticker) => {
-      // bid & ask avg
-      const price = bn(ticker.bid).add(bn(ticker.ask)).div(2);
-      return price.toString();
+      if (ticker.vwap) {
```

Comment:
better to make this configurable so we can still use exchanges that doesn't support this